gfx/portraits/portrait_modifiers/01_headgear.txt

```diff
@@ -3212,9 +3271,17 @@ headgear = {
 			base = 0
 			modifier = {
 				add = 100
-				portrait_wear_helmet_trigger = yes
-				portrait_fp1_norse_clothing_trigger = yes
-				
+				AND = { 
+					portrait_wear_helmet_trigger = yes
+					OR = { 
+						portrait_fp1_norse_clothing_trigger = yes
+						AND = { 
+						# Warcraft
+							portrait_dwarven_clothing_trigger = yes
+							duke_or_highter_clothing_trigger = yes
```

Comment:
Why is it duke or higher if in vanilla it's for common use?

Reply:
We can set it to the Vanilla configuration if you want. The reason I set it differently is because of visual uniqueness. 

For example, the base `northern_war` set uses a nasal helm. 
`fp1_norse_war_common` has a metal coif and eye covering.
`fp1_norse_war` looks very similar, only with gilding. 

Therefore I made this distinction to better identify the social class of characters.

![image](https://user-images.githubusercontent.com/77478662/150826588-e8155277-15fb-433f-9fe2-8f191ebe5293.png)

![image](https://user-images.githubusercontent.com/77478662/150826703-6f48f8cc-4235-4763-ae88-d16e370baf82.png)


![image](https://user-images.githubusercontent.com/77478662/150826656-343705e3-36af-496c-b9f2-edc31030697c.png)

Reply:
Who is the guy who wears `fp1_norse_war_common` on your screenshot?
From what I see your logic is inaccurate. Dukes wear `fp1_norse_war`, not `fp1_norse_war_common`.

Reply:
New commit helmet assignments:

- **`northern_war` (Nasal Helm)(Base CK3)**
  - All dwarves
**`fp1_norse_war_common` (Uncoloured metal coif helmet)(NL dlc)**
  - All dwarves
**`fp1_norse_war` (Gilded metal coif helmet)(NL dlc)**
  - Dukes and higher